app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
Elf/Dwarf classes should also be abstract

Reply:
done :)